streams/client_test.go

```diff
@@ -606,8 +618,11 @@ func getStringFromKinesisStab(s *StubClient, is_gzip bool, t *testing.T) string
 			if err != nil {
 				t.Errorf("unable to read gzipped data error: %v", err)
 			}
-			io.Copy(&buf, zr)
+			var midBuf bytes.Buffer
+			io.Copy(&midBuf, zr)
 			zr.Close()
+			midStr := midBuf.String()
+			buf.WriteString(midStr)
```

Comment:
Did you add the `midBuf` on purpose?

Reply:
yes because the is another buf in the function
gzipBuf sounds better?